R/folders.R

```diff
@@ -0,0 +1,9 @@
+# change subfolder name for raw data cache here to apply globally
+spod_subfolder_raw_data_cache <- function(ver = 1) {
+  rlang:::check_number_whole(ver)
```

Comment:
Nice check function!

Reply:
the use of ::: to its internal functions raises warnings in cmd check, will try to follow the advise of https://josiahparry.com/posts/2024-06-30-type-safety/ to import the source code for these checks

Reply:
#23